.pipelines/templates/.builder-release-template-windows.yaml

```diff
@@ -0,0 +1,139 @@
+      -e VHD_NAME=${VHD_NAME} \
+      ${CONTAINER_IMAGE} make -f packer.mk az-copy
+    displayName: Copying resource to Classic Storage Account
+    condition: and(eq(variables.DRY_RUN, 'False'), eq(variables['MODE'], 'default'))
+  - bash: |
+      echo MODE=$(MODE) && \
+      SA_NAME="$(cat packer-output | grep "storage name:" | cut -d " " -f 3)" && \
+      docker run --rm \
+      -v ${PWD}:/go/src/github.com/Azure/AgentBaker \
+      -w /go/src/github.com/Azure/AgentBaker \
+      -e CLIENT_ID=${BUILD_CLIENT_ID} \
+      -e CLIENT_SECRET="$(BUILD_CLIENT_SECRET)" \
+      -e TENANT_ID=${BUILD_TENANT_ID} \
+      -e SUBSCRIPTION_ID="${BUILD_SUBSCRIPTION_ID}" \
+      -e SA_NAME=${SA_NAME} \
+      -e AZURE_RESOURCE_GROUP_NAME=${BUILD_AZURE_RESOURCE_GROUP_NAME} \
+      ${CONTAINER_IMAGE} make -f packer.mk delete-sa
+    displayName: Clean-up Storage Account
+  - bash: |
+      echo MODE=$(MODE) && \
+      IMAGE_NAME=$SIG_IMAGE_NAME && \
+      docker run --rm \
+      -v ${PWD}:/go/src/github.com/Azure/AgentBaker \
+      -w /go/src/github.com/Azure/AgentBaker \
+      -e CLIENT_ID=${CLIENT_ID} \
+      -e CLIENT_SECRET="$(CLIENT_SECRET)" \
+      -e TENANT_ID=${TENANT_ID} \
+      -e SUBSCRIPTION_ID="${SUBSCRIPTION_ID}" \
+      -e IMAGE_NAME=${IMAGE_NAME} \
+      -e AZURE_RESOURCE_GROUP_NAME=${BUILD_AZURE_RESOURCE_GROUP_NAME} \
+      ${CONTAINER_IMAGE} make -f packer.mk delete-mi
+    displayName: Clean-up Managed Image
+    condition: ne(variables['MODE'], 'default')
+  - bash: |
+      echo MODE=$(MODE) && \
+      OS_DISK_SAS="$(cat packer-output | grep "OSDiskUriReadOnlySas:" | cut -d " " -f 2)" && \
+      VHD_NAME="$(echo $OS_DISK_SAS | cut -d "/" -f 8 | cut -d "?" -f 1)" && \
+      WINDOWS_VERSION="$(grep "OS Version" < release-notes.txt | cut -d ":" -f 2 | tr -d "[:space:]")" && \
+      SKU_NAME="windows-$WINDOWS_SKU" && \
+      docker run --rm \
+      -v ${PWD}:/go/src/github.com/Azure/AgentBaker \
+      -w /go/src/github.com/Azure/AgentBaker \
+      -e CLIENT_ID=${CLIENT_ID} \
+      -e CLIENT_SECRET="$(CLIENT_SECRET)" \
+      -e TENANT_ID=${TENANT_ID} \
+      -e SUBSCRIPTION_ID="${SUBSCRIPTION_ID}" \
+      -e CLASSIC_SA_CONNECTION_STRING="$(CLASSIC_SA_CONNECTION_STRING)" \
+      -e STORAGE_ACCT_BLOB_URL=${STORAGE_ACCT_BLOB_URL} \
+      -e VHD_NAME=${VHD_NAME} \
+      -e OS_NAME="Windows" \
+      -e SKU_NAME=${SKU_NAME} \
+      -e OFFER_NAME="Windows" \
+      -e IMAGE_VERSION=${WINDOWS_VERSION} \
+      -e HYPERV_GENERATION="V1" \
+      ${CONTAINER_IMAGE} make -f packer.mk generate-sas
+    displayName: Getting Shared Access Signature URI
+    condition: and(eq(variables.DRY_RUN, 'False'), eq(variables['MODE'], 'default'))
+  - task: PublishPipelineArtifact@1
+    inputs:
+        artifactName: 'publishing-info-${{ parameters.artifactName }}'
+        targetPath: 'vhd-publishing-info.json'
+    condition: and(eq(variables.DRY_RUN, 'False'), ne(variables['MODE'], 'sigMode'))
```

Comment:
Is this for PIR? We also need to support SIG for production soon.

Reply:
SIG mode means people build images to SIG by specifying sig name and image definition etc. sigMode is not for production.

Reply:
So is it not used for AKS SIG?

Reply:
AKS SIG is outside of building VHD. This sigMod is to bring your own SIG image for testing.